src/kibana/index.html

```diff
@@ -16,8 +16,8 @@
       <center>
         <img width="128" alt="Loading Kibana" src="images/initial_load.gif">
         <h1>
-          <strong>Kibana</strong>
-          <small id="cache-message">is loading. Give me a moment here. I'm loading a whole bunch of code. Don't worry, all this good stuff will be cached up for next time!</small>
+          <strong>NetMon</strong>
+          <small id="cache-message">is loading. Your results will be available momentarily!</small>
```

Comment:
momentarily can mean two things:
1. In a short while
2. For only a little bit of time